qiskit/primitives/backend_sampler.py

```diff
@@ -0,0 +1,234 @@
+    any measurement mitigation, it just computes the probability distribution
+    from the counts.
+    """
+
+    def __init__(
+        self,
+        backend: "Backend" = None,
+        circuits: "QuantumCircuit" | Iterable["QuantumCircuit"] = None,
+        parameters: Iterable[Iterable["Parameter"]] | None = None,
+        bound_pass_manager: "PassManager" | None = None,
+        skip_transpilation: bool = False,
+    ):
+        """Initialize a new BackendSampler
+
+        Args:
+            backend: Required: the backend to run the sampler primitive on
+            circuits: The Quantum circuits to be executed.
+            parameters: Parameters of each of the quantum circuits.
+                Defaults to ``[circ.parameters for circ in circuits]``.
+            bound_pass_manager: An optional pass manager to run after
+                parameter binding.
+            skip_transpilation: If this is set to True the internal compilation
+                of the input circuits is skipped and the circuit objects
+                will be directly executed when this objected is called.
+        Raises:
+            ValueError: If backend is not provided
+        """
+
+        super().__init__(circuits, parameters)
+        if backend is None:
+            raise ValueError("A backend is required to use BackendSampler")
+        self._backend = backend
+        self._run_options = self._backend.options
+        self._is_closed = False
+        self._transpile_options = Options()
+        self._bound_pass_manager = bound_pass_manager
+        self._preprocessed_circuits: list["QuantumCircuit"] | None = None
+        self._transpiled_circuits: list["QuantumCircuit"] | None = None
+        self._skip_transpilation = skip_transpilation
+
+    @property
+    def preprocessed_circuits(self) -> list["QuantumCircuit"]:
+        """
+        Preprocessed quantum circuits produced by preprocessing
+        Returns:
+            List of the transpiled quantum circuit
+        Raises:
+            QiskitError: if the instance has been closed.
+        """
+        self._check_is_closed()
+        return list(self._circuits)
+
+    @property
+    def transpiled_circuits(self) -> list["QuantumCircuit"]:
+        """
+        Transpiled quantum circuits.
+        Returns:
+            List of the transpiled quantum circuit
+        Raises:
+            QiskitError: if the instance has been closed.
+        """
+        self._check_is_closed()
+        if self._skip_transpilation:
+            self._transpiled_circuits = list(self._circuits)
+        elif self._transpiled_circuits is None:
+            # Only transpile if have not done so yet
+            self._transpile()
+        return self._transpiled_circuits
+
+    @property
+    def backend(self) -> "Backend":
+        """
+        Returns:
+            The backend which this sampler object based on
+        """
+        return self._backend
+
+    @property
+    def run_options(self) -> Options:
```

Comment:
`run_options` is supported in BaseClass and the name will be `options` https://github.com/Qiskit/qiskit-terra/pull/8719.